src/Ledger/Utxo/Properties.lagda

```diff
@@ -91,39 +104,42 @@ consumedCoinEquality : ∀ {pp} → coin (mint tx) ≡ 0
                      → coin (consumed pp utxoState tx)
                      ≡ cbalance ((UTxOState.utxo utxoState) ∣ txins tx) + depositRefunds pp utxoState tx
 consumedCoinEquality {tx} {utxoState} {pp} h = let utxo = UTxOState.utxo utxoState in begin
-  coin (balance (utxo ∣ txins tx) +ᵛ mint tx +ᵛ inject (depositRefunds pp utxoState tx))
+  coin (balance (utxo ∣ txins tx) + mint tx + inject (depositRefunds pp utxoState tx))
     ≡⟨ ∙-homo-Coin _ _ ⟩
-  coin (balance (utxo ∣ txins tx) +ᵛ mint tx) + coin (inject $ depositRefunds pp utxoState tx)
-    ≡⟨ cong (coin (balance (utxo ∣ txins tx) +ᵛ mint tx) +_) (property _) ⟩
-  coin (balance (utxo ∣ txins tx) +ᵛ mint tx) + depositRefunds pp utxoState tx
-    ≡⟨ cong! (∙-homo-Coin _ _) ⟩
```

Comment:
This is one of the proof steps that (I think) must be re-written in order to use the overloaded `_+_` operator symbol. If someone know how to get tactics and instances to work well together in this situation, please explain.